### PR TITLE
new copy on home page

### PR DESCRIPTION
**Pull request by pflopez**
Screen Shot 2023-09-29 at 12 48 29

- Fixes scroll so the top nav bar is fixed, box centered and scrollable if does not fit 
- Also fixes the scroll on planning areas

### CODE REVIEW

File: src/interface/src/app/home/welcome/welcome.component.scss [hunk 1]
```diff
@@ -26,8 +37,14 @@ p {
 .flex {
   display: flex;
   align-items: flex-start;
+  gap:20px;
```
Comment (kinstella) [inline src/interface/src/app/home/welcome/welcome.component.scss:40]: This makes me curious: do we enforce linting conventions for CSS/SCSS? Should we? 
I'm fine either way, just was surprised to see no space here.

Comment (lastminutediorama) [inline src/interface/src/app/home/welcome/welcome.component.scss:40]: This lack of space made me curious if we have a convention for linting CSS/SCSS -- I'm ok with having one or not having one, either way.

**Review by lastminutediorama — APPROVED** [review]
thanks for this, looks like a nice improvement

File: src/interface/src/app/home/welcome/welcome.component.scss (hunk 1 above)
Comment (pflopez) [reply src/interface/src/app/home/welcome/welcome.component.scss:40]: Good point, we should! Lemme see if we can use prettier for this as well

**Pull request merged**